### ISSUE TITLE
Pack not signing

### ISSUE THREAD

**Issue by AdlerSalbei** [BrettMayson/HEMTT#320]
**HEMTT Version:** `0.7.6`
**Project:** https://github.com/gruppe-adler/grad_trenches

**Description:**

Using the pack command, it does not sign the addon file.

**Steps to reproduce:**

1. Use pack command.
2. No signed file in release

**Comment by Dahlgren** [comment]
Use `hemtt build --release` creates both bisign and keys for me.

```
$ hemtt build --release
HEMTT 0.7.6
Environment: dev

Clean
Checks         [00:00:00] [##############################] [3|3]
Prebuild       [00:00:00] [##############################] [0|0]
Build          [00:00:00] [##############################] [0|0]
Release
Sign
ReleaseBuild `@zip grad_trenches_1.5.9`
Archiving grad_trenches_1.5.9.zip

$ dir /b releases\1.5.9.4\@grad_trenches\addons
grad_trenches_assets.pbo
grad_trenches_assets.pbo.grad_trenches_1.5.9.4-2aa9f896.bisign
grad_trenches_functions.pbo
grad_trenches_functions.pbo.grad_trenches_1.5.9.4-2aa9f896.bisign
grad_trenches_main.pbo
grad_trenches_main.pbo.grad_trenches_1.5.9.4-2aa9f896.bisign

$ dir /b releases\1.5.9.4\@grad_trenches\keys
grad_trenches_1.5.9.4.bikey
```

**Comment by synixebrett** [comment]
This bug is related to the `hemtt pack --release` command

**Comment by BrettMayson** [comment]
Is there a reason to build a release without binarize? If you need to skip binarization of a file that is now possible with 1.0

**Comment by BrettMayson** [comment]
Closed with the release of 1.0.0. If you feel this still needs attention for 1.0.0, please re-open.
